docs/capabilities.md

```diff
@@ -56,6 +56,7 @@ Capability | Description
 |`appium:showXcodeLog`|Whether to display the output of the Xcode command used to run the tests. If this is `true`, there will be **lots** of extra logging at startup. Defaults to `false`|e.g., `true`|
 |`appium:iosInstallPause`|Time in milliseconds to pause between installing the application and starting `WebDriverAgent` on the device. Used particularly for larger applications. Defaults to `0`|e.g., `8000`|
 |`appium:usePrebuiltWDA`|Skips the build phase of running the WDA app. Building is then the responsibility of the user. Only works for Xcode 8+. Defaults to `false`.|e.g., `true`|
+|`appium:usePreinstalledWDA`| Whether to launch a preinstalled WebDriverAgentRunner application instead of running `xcodebuild` to start a WebDriverAgentRunner process for real devices. The preinstalled WebDriverAgent package must have been built by Xcode 12+. The default target bundle identifier is `com.facebook.WebDriverAgentRunner.xctrunner`. If `appium:updatedWDABundleId` is provided, `<appium:updatedWDABundleId>.xctrunner` will be launched. Please read [Run Preinstalled WebDriverAgentRunner](./run-preinstalled-wda.md) for more details. Defaults to `false`. |`true` or `true`|
```

Comment:
preinstalled WebDriverAgentRunner application using a custom XCTest API client instead

Reply:
`to start a WebDriverAgentRunner process` is redundant

Reply:
`true` or `true` -> typo?